tests/Tests.Json.Decode.fs

```diff
@@ -1162,14 +1154,14 @@ Expecting an object but instead got:
                 let expected =
                     Error(
                         """
-Error at: `$.version`
+Error at: `$`
 Expecting an object with a field named `version` but instead got:
 {
     "info": 3,
     "data": 2
 }
                         """.Trim())
-                let infoHelp version : Decode.Decoder<int> =
+                let infoHelp version : Decoder<int> =
```

Comment:
Because we change the path for accessing the types API this is a breaking change.

Especially, because before users where forced to do `Decode.Decoder<'T>` so all the old code will break.

Reply:
Yeah, maybe it could be done in another way. But the easies option was to move `Decoder<'T>` and `Encode<'T>` to the common Types file, so we could have the cached extra decoders.